lib/errors.js

```diff
@@ -71,6 +71,13 @@ ValidationError.serializers = {
     `Invalid value of a enum in a property '${error.property}' ` +
     `allowed: ${wrap(error.info.expected)}, actual: '${error.info.actual}'`,
   undefinedEntity: error => `Undefined ${error.info} '${error.property}'`,
+  invalidAddProp: error =>
```

Comment:
Why `Add` though? Perhaps just `invalidProperty` (considering we already have `unresolvedProperty`, `missingProperty`)?

Reply:
`invalidAddProp` is for Invalid Additional Property, as it is only reported in cases of properties that are not defined in a schema, thus `invalidProperty` is not the best fit.
Though I don't like `invalidAddProp` either.

Reply:
Maybe better name is `forbiddenAdditionalProp` or `notAllowedAdditionalProperty`?